backend/src/main/kotlin/pl/poznan/put/tsd/planningpoker/backend/resources/MessageHandler.kt

```diff
@@ -0,0 +1,125 @@
+package pl.poznan.put.tsd.planningpoker.backend.resources
+
+import arrow.core.Either
+import arrow.core.raise.either
+import arrow.core.raise.ensure
+import arrow.core.raise.ensureNotNull
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import kotlinx.coroutines.runBlocking
+import org.springframework.context.annotation.Configuration
+import org.springframework.stereotype.Component
+import org.springframework.web.socket.CloseStatus
+import org.springframework.web.socket.TextMessage
+import org.springframework.web.socket.WebSocketSession
+import org.springframework.web.socket.config.annotation.EnableWebSocket
+import org.springframework.web.socket.config.annotation.WebSocketConfigurer
+import org.springframework.web.socket.config.annotation.WebSocketHandlerRegistry
+import org.springframework.web.socket.handler.TextWebSocketHandler
+import pl.poznan.put.tsd.planningpoker.backend.model.Game
+import pl.poznan.put.tsd.planningpoker.backend.resources.responses.GameResponse.Companion.toResponseModel
+import pl.poznan.put.tsd.planningpoker.backend.resources.responses.PlayerResponse.Companion.toResponseModel
+import pl.poznan.put.tsd.planningpoker.backend.services.GamesService
+import java.io.IOException
+import java.lang.IllegalArgumentException
+import java.net.URI
+import java.util.UUID
+
+// https://codemwnci.medium.com/kotlin-springboot-and-websockets-276029b22482
+
+enum class MessageType {
+    Error,
+    Game
+}
+
+
+@Component
+class MessageHandler(val gamesService: GamesService) :
+    TextWebSocketHandler() {
+    private class Message(val type: MessageType, val data: Any)
+
+    private val mapper = jacksonObjectMapper()
```

Comment:
for testing purposes it might be better to have it wired through dependency injection, but just a nitpick :D